### PR TITLE
civo: Add version 0.6.42

### PR DESCRIPTION
**Pull request by jaysonsantos**
As this is not really that popular I didn't put on main repo, let me know if it should change.

### CODE REVIEW

**Comment by jaysonsantos** [comment]
Hey there @Ash258 could you point out what needs fix?

File: bucket/civo.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    },
+    "checkver": {
+        "github": "https://github.com/civo/cli"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/civo/cli/releases/download/v$version/civo-$version-windows-amd64.zip"
+            }
+        }
```
Comment (Ash258) [inline bucket/civo.json:21]:
```suggestion
        },
        "hash": {
            "url": "$baseurl/civo-$version-checksums.sha256"
        }
```

File: bucket/civo.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "bin": "civo.exe",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/civo/cli/releases/download/v0.6.42/civo-0.6.42-windows-amd64.zip",
+            "hash": "a63385c8dc27363177afd6b1946a8a530413b11a52935b40f5c8b6a95b827dbd"
+        }
+    },
+    "checkver": {
+        "github": "https://github.com/civo/cli"
+    },
```
Comment (Ash258) [inline bucket/civo.json:15]:
```suggestion
    "bin": "civo.exe",
    "checkver": "github",
```

File: bucket/civo.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+    "version": "0.6.42",
+    "description": "Command Line Interface (CLI) for interacting with your Civo resources",
+    "homepage": "https://github.com/civo/cli",
+    "license": "Apache-2.0",
+    "bin": "civo.exe",
```
Comment (Ash258) [inline bucket/civo.json:6]:
```suggestion
```

File: bucket/civo.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+    "version": "0.6.42",
+    "description": "Command Line Interface (CLI) for interacting with your Civo resources",
```
Comment (Ash258) [inline bucket/civo.json:3]:
```suggestion
    "description": "CLI library for managing cloud resources such as instances and Kubernetes clusters at Civo.com.",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Comment by jaysonsantos** [comment]
Should I rebase it or will you squash merge it?

**Review by Ash258 — APPROVED** [review]

**Pull request merged**